docs/gadgets/profile/tcprtt.md

```diff
@@ -0,0 +1,8 @@
+---
+title: 'Using profile tcprtt'
+weight: 20
+description: >
+  Analyze TCP connections through an RTT (Round-Trip Time) distribution
+---
+
+TODO: write this guide
```

Comment:
:see_no_evil:

Reply:
I'm working on this. It will be identical to what you did for tcpconnect's latency flag but explaining that it represents a different latency. I'm reading about this difference to make it clear.